web/ui/module/codemirror-promql/src/complete/promql.terms.ts

```diff
@@ -142,12 +142,14 @@ export const functionIdentifierTerms = [
     detail: 'function',
     info: 'Calculate the cosine, in radians, for input series',
     type: 'function',
+    boost: -1,
```

Comment:
Could you add comments like:

```
// Avoid ranking higher than `count`.
...

// Avoid ranking higher than `delta`.
...

// Avoid ranking higher than `rate`.
...
```

That way a reader later on understands easily why this is here.

Reply:
good idea !
added